frontstage/views/surveys/help/surveys_help.py

```diff
@@ -393,6 +290,58 @@ def send_help_message(session, survey_ref, ru_ref):
         return redirect(url_for("surveys_bp.get_survey_list", tag="todo"))
 
 
+def abort_help_if_session_not_set():
+    """
+    Aborts the process if flask session attributes are not set
+    """
+    if flask_session["help_survey_ref"] is None or flask_session["help_ru_ref"] is None:
+        abort(404)
+
+
+def get_selected_survey_business_details():
```

Comment:
Can you change this to:
```python
def get_selected_survey_business_details() -> tuple[str, str, str, str, str]:
```
Also, a function returning 5 values is...uncommon.  It's probably fine, but maybe it should  be a dict when it's that many values?  I don't know what the best practice is around this

Reply:
I might be right.. but I also don't know whats the best practice . I have updated the return type.